scripts/collectIncludes.gradle

```diff
@@ -21,13 +40,19 @@ task collectIncludes(
                 if (!collections[type]) {
                     collections[type] = []
                 }
-                collections[type] << (file.canonicalPath - projectDir.canonicalPath)[1..-1]
+                println "file: " + file.canonicalPath
```

Comment:
ok. this is only debug output, isn't it?
we should turn this into `logger.info`-statements, shouldn't we?